src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for a Printable contact detail for a Person in the addressbook.
```

Comment:
good to see you add comments here. however, think if this interface is particular to person details?
any class implementing this interface provides the printable string format, no?
so, you can make the comment little more specific than this.